### PR TITLE
Adding more troubleshooting

### PR DESCRIPTION
**Pull request by digininja**
I've added a couple more troubleshooting entries to the README to cover the standard "the database won't connect" and trying to get MySQL 8 to work with PHP.

This relies on the PR <https://github.com/ethicalhack3r/DVWA/pull/357> which shows the actual error in the setup script.

There are probably more things to add to this, but it is a start.

### CODE REVIEW

**Comment by digininja** [comment]
I think this may be ready to be merged. If anyone can suggest any other auth problems that need adding, let me know and I'll get them added.

File: README.md [hunk 1]
```diff
@@ -34,19 +34,16 @@ You should have received a copy of the GNU General Public License
...
-- [dockerhub page](https://hub.docker.com/r/vulnerables/web-dvwa/)
-`docker run --rm -it -p 80:80 vulnerables/web-dvwa`
-
-Please ensure you are using aufs due to previous MySQL issues. Run `docker info` to check your storage driver. If it isn't aufs, please change it as such. There are guides for each operating system on how to do that, but they're quite different so we won't cover that here.
 
 ## Download
 
-DVWA is available either as a package that will run on your own web server or as a Live CD:
+While there are various versions of DVWA around, the only supported version is the latest source from the offical Github repository. You can either lone it from the repo:
+
```
Comment (ethicalhack3r) [inline README.md:41]: Typo `lone` should be `clone`

Comment (digininja) [reply README.md]: The GitHub built in markdown editor doesn't do spell checking, that's annoying. I've fixed a few typos.

**Comment by ethicalhack3r** [comment]
Is this ready to merge?

**Comment by digininja** [comment]
Ye, go on. There are probably a few bits that will need updating and more
adding, but it has the main issues covered in the troubleshooting so
hopefully it will help fix the majority of problems before they become
issues.

**Pull request merged**